s4e-backend/src/main/java/pl/cyfronet/s4e/data/repository/SceneRepositoryImpl.java

```diff
@@ -0,0 +1,352 @@
+package pl.cyfronet.s4e.data.repository;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.jdbc.core.JdbcTemplate;
+import pl.cyfronet.s4e.bean.Product;
+import pl.cyfronet.s4e.bean.Scene;
+import pl.cyfronet.s4e.properties.DbProperties;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Map;
+
+@Slf4j
+public class SceneRepositoryImpl implements SceneRepositoryExt {
```

Comment:
If class contain more than 120 lines or can not feet in IDE window, mean that have more than 1 responsibility.

Reply:
now should be better :)